lib/LaTeXML/Core/Definition/CharDef.pm

```diff
@@ -47,13 +47,17 @@ sub invoke {
   my $mathglyph = $$self{mathglyph};
   # A dilemma: If the \chardef were in a style file, you're prefer to revert to the $cs
   # but if defined in the document source, better to use \char ###\relax, so it still "works"
-  if (defined $mathglyph) {    # Must be a math char
+  my $src   = $$self{locator} && $$self{locator}->toString;
+  my $local = $src && $src !~ /\.(?:sty|ltxml|ltxmlc)/;    # Dumps currently have undefined src!
```

Comment:
Is `.ltxmlc` a new extension? We don't have them in the repository at the moment. If it's dump related, why `c` and not `d`? Or even better `.ltxml.dump` or `.dump.ltxml`

Reply:
Exactly for dumps and your naming suggestion sounds good